libs/wingsdk/src/cloud/function.ts

```diff
@@ -40,6 +40,12 @@ export interface FunctionProps {
    * @default 30
    */
   readonly logRetentionDays?: number;
+
+  /**
+   * The maximum concurrent invocations that can run at one time.
+   * @default - platform specific limits
```

Comment:
maybe worth mentioning the default in the simulator here?

Reply:
OK